### PR TITLE
style(less): Less Hint Exceptions

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-716

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM
- [x] ~~QE LGTM~~ (no JS updated)

### CODE REVIEW

File: .lesshint [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "enabled": false
     },
     "importantRule": {
-        "enabled": false // TODO: enable on audit of !important
+        "enabled": true // TODO: enable on audit of !important
```
Comment (glynnis) [inline .lesshint]: Should we remove the TODO here?

**Comment by glynnis** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

File: src/components/rxMultiSelect/rxMultiSelect.less [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * rxMultiSelect
  */
 rx-multi-select {
-  z-index: auto !important;
+  z-index: auto;
```
Comment (100stacks) [inline src/components/rxMultiSelect/rxMultiSelect.less]: We also remove in PR #1594 - https://github.com/100stacks/encore-ui/blob/FRMW-641-z-index-fix-rxMultiSelect/src/components/rxMultiSelect/rxMultiSelect.less#L6 :+1:

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**